setup.py

```diff
@@ -75,9 +66,8 @@
     'invenio-oauthclient>=1.0.0',
     'invenio-userprofiles>=1.0.0',
     'invenio-query-parser>=0.3.0',
-    'invenio[{db},{es},base,auth,metadata]~={version}'.format(
-        db=DATABASE, es=ELASTICSEARCH, version=INVENIO_VERSION
-    ),
+    'invenio[auth,base,metadata,postgresql]==3.0.0',
+    'invenio-db[postgresql,versioning]==1.0.1',
```

Comment:
why is this needed? it is not added with `invenio[postgresql]`?

Reply:
I remember there was some error because `invenio[postgresql]` indeed installs `invenio-db[postgresql,versioning]` but with this constraint `'>=1.0.9,<1.1.0'` and we already have `'invenio-db[postgresql,versioning]==1.0.1'` in `requirements.txt` so I added in setup.py to be explicit.